### ISSUE TITLE
Option to disable alt keyboard shortcuts

### ISSUE THREAD

**Issue by armiller** [glowing-bear/glowing-bear#572]
I use alt + [0-9] to switch browser tabs regularly in Chrome. Once I land on the glowing bear tab I'm stuck switching between weechat buffers. It would be nice to have an option to disable any alt keybinding so a user can switch back and forth between tabs without using a mouse or alternative shortcut.

**Comment by Nicd** [comment]
This would be a good feature also because currently it is not possible to type an @ or $ or £ or any of the other special characters that are behind the alt (option) key on a MacBook. GB catches both alt keys as the buffer change shortcut.

**Comment by anders** [comment]
in the OS X Terminal ESC can be used as a replacement for the meta key, would be nice if that would be the case in glowing bear as well

**Comment by lorenzhs** [comment]
I'm having some trouble with keyboard shortcuts as well because Chrome just doesn't handle my keyboard layout correctly. It's frustrating as hell (I checked, it sends the wrong events). Since all browsers do this in different subtly broken ways, it's likely out of scope for us to handle this. If anyone has any suggestions for *lightweight* libraries that handle this in a configurable matter (i.e., to enable this usecase), that would be much appreciated.

**Comment by Nicd** [comment]
I think a good start would be to have an option to disable the alt key buffer switcher. If you don't suppress the key event, people should be able to type normally.
